tests/conftest.py

```diff
@@ -93,6 +93,7 @@
     # Currently there is a runtime incompatibility caused by the library:
     # `undefined symbol: _PyInterpreterState_Get`
     collect_ignore_glob.append("*test_psycopg2*")
+    collect_ignore_glob.append("*test_pep0249*")
```

Comment:
We don't want to ignore the pep 249 tests after this PR gets merged right? You can safely remove this line.

Reply:
Done.

Reply:
Unfortunately we have a dependency on psycopg2 package, it gives `undefined symbol: _PyInterpreterState_Get` error. It'll be solved when the Python version 3.13 released.

Reply:
Does it work using `sqlite` instead of `psycopg2`?